common/src/main/scala/explore/cache/CacheComponent.scala

```diff
@@ -6,37 +6,36 @@ package explore.cache
 import cats.effect.Resource
 import cats.effect.kernel.Deferred
 import cats.effect.syntax.all.given
-import cats.syntax.all.given
+import cats.syntax.all.*
 import crystal.react.View
 import crystal.react.hooks.*
 import fs2.concurrent.SignallingRef
 import japgolly.scalajs.react.*
 import japgolly.scalajs.react.feature.Context
 import japgolly.scalajs.react.util.DefaultEffects.{Async => DefaultA}
+import japgolly.scalajs.react.util.DefaultEffects.{Sync => DefaultS}
 import japgolly.scalajs.react.vdom.html_<^.*
 import lucuma.ui.syntax.pot.*
 
-trait CacheComponent[Props: Reusability, A]:
-  private type F[T] = DefaultA[T]
+import scala.tools.nsc.doc.model.Def
```

Comment:
Is this being used? Or was metals being helpful?

Reply:
Metals, definitely